chat/chat-tool/tool/src/webapp/jsp/editRoom.jsp

```diff
@@ -82,19 +83,17 @@
             <h:panelGroup styleClass="chat-date-instructions">
               <sakai:instruction_message value="#{msgs.custom_date_instructions}" />
             </h:panelGroup>
-            <h:panelGroup styleClass="chat-select-date longtext indnt1">
-              <h:outputLabel value="#{msgs.custom_date_start} #{msgs.custom_date_entry_format_description}" for="startDate" styleClass="chat-date-label"/>
-              <t:inputCalendar id="startDate" value="#{ChatTool.currentChannelEdit.startDate}" renderAsPopup="true" renderPopupButtonAsImage="true"
-                popupTodayString="#{msgs.custom_date_entry_today_is}" popupWeekString="#{msgs.custom_date_entry_week_header}"
-                popupDateFormat="#{msgs.custom_date_entry_format}" popupGotoString="#{msgs.custom_date_entry_goto}" />
-              <h:message for="startDate" styleClass="alertMessageInline" />
+			<h:panelGroup styleClass="longtext indnt1 ">
+				<h:message for="startDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br></h:message>
+				<h:message for="endDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br/></h:message>
```

Comment:
Spacing is inconsistent here. Also, inline styles are not great practise. I would be better to offload these into an appropriate CSS file.